abb_experimental.rosinstall

```diff
@@ -0,0 +1,5 @@
+#Temporary until PR accepted by https://github.com/ros-industrial/
+- git: {local-name: abb, uri: 'https://github.com/AustinDeric/abb.git', version: kinetic-devel}
+
+- git: {local-name: industrial_core, uri: 'https://github.com/ros-industrial/industrial_core.git', version: kinetic-devel}
```

Comment:
Same comment as in ros-industrial/abb#126: `industrial_core` has a Kinetic release, so if possible could we use that?